control/controlclient/hostinfo_linux.go

```diff
@@ -145,3 +148,11 @@ func inHerokuDyno() bool {
 	}
 	return false
 }
+
+func inAzure() (ret bool) {
```

Comment:
leftover debugging named return value?

Reply:
and be more specific about which azure service?

Reply:
Done.